### PR TITLE
move funcs from test/e2e/framework/job to test/e2e/apps

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

move funcs from test/e2e/framework/job to test/e2e/apps and test/e2e/node
remove unused code func NewMetricExporter

**Which issue(s) this PR fixes**:

ref:https://github.com/kubernetes/kubernetes/issues/86052

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/framework/autoscaling/autoscaling_utils.go [hunk 1]
```diff
@@ -112,12 +112,6 @@ func NewDynamicResourceConsumer(name, nsName string, kind schema.GroupVersionKin
 		dynamicRequestSizeInMillicores, dynamicRequestSizeInMegabytes, dynamicRequestSizeCustomMetric, cpuLimit, memLimit, clientset, scaleClient, nil, nil)
 }
 
-// NewMetricExporter is a wrapper to create a new ResourceConsumer for metrics exporter
```
Comment (tanjunchen) [inline test/e2e/framework/autoscaling/autoscaling_utils.go]: the  unused  functions , so delete them .

File: test/e2e/apps/job.go [hunk 2]
```diff
@@ -18,11 +18,14 @@ package apps
 
 import (
 	"fmt"
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
```
Comment (oomichi) [inline test/e2e/apps/job.go]: This import line block seems for general libraries.
It is better to move the above lines to the import block of line 25 for k8s.io.

File: test/e2e/node/ttlafterfinished.go [hunk 3]
```diff
@@ -17,19 +17,27 @@ limitations under the License.
 package node
 
 import (
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (oomichi) [inline test/e2e/node/ttlafterfinished.go]: ditto

File: test/e2e/apps/job.go (hunk 2 above)
Comment (tanjunchen) [reply test/e2e/apps/job.go]: Thank you. before I thought running hack/update-gofmt.sh would work. i will update it

**Comment by tanjunchen** [comment]
/cc @oomichi 
done

**Review by oomichi — COMMENTED** [review]
Thanks for updating

/lgtm

**Review by SataQiu — COMMENTED** [review]
Thanks @tanjunchen 
/approve

**Pull request merged**